security/cmd/chiron/demo-deploy/protomutate-webhook.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  labels:
+    app: protomutateWebhook
+    chart: protomutateWebhook
+    heritage: Tiller
+    release: istio
+  name: protomutate
+webhooks:
+- clientConfig:
+    caBundle: ""
+    service:
+      name: protomutate
```

Comment:
what's this service?

Reply:
It is a webhook service for demo Istio Webhook Controller.